flytekit/core/checkpointer.py

```diff
@@ -102,7 +102,8 @@ def restore(self, path: typing.Optional[typing.Union[Path, str]] = None) -> typi
         if path is None:
             p = Path(self._td.name)
             path = p.joinpath(self.SRC_LOCAL_FOLDER)
-            path.mkdir()
+            if not path.exists():
+                path.mkdir()
```

Comment:
We can pass [`exist_ok`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir) instead.

Reply:
Thanks @eapolinario , changed this in 33f2d6ba